### DISCUSSION TITLE
SonarCloud 연동

### DISCUSSION THREAD

**Discussion by nitronium102** [EFUB-LakkuLakku/LakkuLakku-Back#107] [General]
SonarQube는 직접 웹서버에 세팅을 해야 하는데 SonarCloud는 클라우드 서비스라 그럴 필요 X
https://zion830.tistory.com/79

그리고 IntelliJ 안에서 sonarLint 플러그인 사용하면 PR 올리기 전에도 본인이 알아서 수정할 수 있습니다!

**Comment by june0216** [comment]
감사합니다!

**Comment by SeojungH** [comment]
감사합니다!!
